lib/registration.pm

```diff
@@ -68,7 +68,8 @@ our %SLE15_DEFAULT_MODULES = (
     sles4sap => 'base,desktop,serverapp,ha,sapapp',
 );
 
-our @SLE15_ADDONS_WITHOUT_LICENSE = qw(ha sdk wsm we hpcm);
+our @SLE15_ADDONS_WITHOUT_LICENSE        = qw(ha sdk wsm we hpcm);
```

Comment:
what if 
```suggestion
our @SLE15_ADDONS_WITHOUT_LICENSE        = qw(sdk wsm hpcm);
push @addons_with_license, qw(ha we) unless get_var('IN_PATCH_SLE');
```
and you don't need all other code. Or I missing something ?

Reply:
yeah , it is not so easy ...

Reply:
Hmm, I like this, but I'm not sure if that `push` will be evaluated before or after IN_PATCH_SLE is set to 1 since it's outside the method being called.